app/bot/when.go

```diff
@@ -0,0 +1,134 @@
+package bot
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+)
+
+// When bot is answer on question "when the stream is started".
+type When struct {
+	now     func() time.Time
+	reactOn []string
+}
+
+// NewWhen makes a new When bot.
+func NewWhen(reactOn []string, nowFunc func() time.Time) *When {
```

Comment:
this constructor is too detailed for the caller. Pls make reacton to be a part of the bot itself (local const/var) and set now to the proper default (time.Now)

Reply:
fixed